docs/database.md

```diff
@@ -0,0 +1,51 @@
+# Connect to database
+
+## local
+
+1. `psql -h localhost -p 5678 -U prisma -W`
+2. enter password `prisma`
+3. run `set search_path to "default$default";`
+
+## production
+
+1. in a terminal window run
+`cloud_sql_proxy -instances=moocfi:europe-north1:quizzes-backend=tcp:0.0.0.0:3306`
```

Comment:
`cloud_sql_proxy` has to be installed separately; an alternative to running the proxy every time is to add your IP range to the Google Cloud Console (SQL -> Connections -> Authorized networks)

Reply:
Another good point, added installation instructions for `cloud_sql_proxy` and mentioned the alternative